### ISSUE TITLE
No syntax highlighting for monkey2 files accepted by mx2cc

### ISSUE THREAD

**Issue by forgotten-king** [engor/Ted2Go#61]
The syntax highlighting seems to only work for files with `.monkey2` extension. Though, this is inconsistent with the monkey2 extensions mx2cc accepts. Possibly just ignore the case when choosing a monkey file to parse or add the other reasonable `.Monkey2` extension?
I believe *most* of the related files are here: [/syntax/Monkey2Highlighter.monkey2](https://github.com/engor/Ted2Go/blob/master/syntax/Monkey2Highlighter.monkey2)

tldr; I can build `.Monkey2` files, but syntax highlighting in Ted2Go only works with `.monkey2`.

**Comment by engor** [comment]
Fixed, please check it out.

**Comment by forgotten-king** [comment]
Looks like that probably got the job fixed! I'll have to wait until the next released version of monkey for now, but nice and speedy work friend!
